core/src/test/java/apoc/export/csv/ExportCsvTest.java

```diff
@@ -380,6 +381,23 @@ public void testExportQueryCsvIssue1188() throws Exception {
 
     }
 
+    @Test
+    public void testExportWgsPoint() {
+        db.executeTransactionally("CREATE (p:Position {place: point({latitude: 12.78, longitude: 56.7, height: 1.1})})");
+
+        TestUtil.testCall(db, "CALL apoc.export.csv.query($query, null, {quotes: 'none', stream: true}) YIELD data RETURN data",
+                map("query", "MATCH (p:Position) RETURN p.place as place"),
+                (r) -> {
+                    String data = (String) r.get("data");
+                    Map<String, Object> place = Util.fromJson(data.split(System.lineSeparator())[1], Map.class);
+                    assertEquals(12.78D, place.get("latitude"));
```

Comment:
I think `assertEquals(double,double)` is deprecated
`assert(double,double,double)` should be used (the last parameter is the delta)